rfcs/0005-rfc-process.md

```diff
@@ -0,0 +1,54 @@
+---
+created-on: 2021-03-29
+freshness: 2021-03-29
+owner: @dj2
+status: draft
+---
+
+[La version française suit.](#résumé)
+
+# RFC Process
+
+## Summary
+This RFC lays out some foundations for the RFC process, mostly around
+the approval and acceptance process.
+
+## Motivation
+Without some sort of framework it is difficult to know how the RFC
+process is going to work and to align everyone in the same direction.
+There are questions around when approvals are needed, how many, how long
+do we wait. This RFC attempts to set some guard rails for this process
+and to start us in the right direction.
+
+## Proposal
```

Comment:
Thanks Dan, looks good! I'm wondering if we want to provide an `rfc_template.md` for an RFC, or just provide a basic guideline for consistency? E.g. sections like:
- Title
- Objective
- Motivation
- Proposal 
- ... other optional sections?

Reply:
There is a template provided, it gives a bit of advice on what to put in each section. I'll link to it from this RFC so the things are connected. (https://github.com/cds-snc/docs/blob/main/rfc-template.md)

Reply:
Perfect, the link is great!